pkg/scaffold/v2/types.go

```diff
@@ -75,6 +75,9 @@ type {{.Resource.Kind}}Status struct {
 }
 
 // +kubebuilder:object:root=true
+{{ if not .Resource.Namespaced }}
+// +kubebuilder:resource:scope=Cluster
```

Comment:
Per https://github.com/kubernetes-sigs/controller-tools/blob/4d432d9dbc86bbf594c03f853878af64df8b95ec/pkg/crd/markers/crd.go#L181-L187, IIUC `path` should present as well. Otherwise, `make manifests` will complain.

Reply:
I think we'll just want to make that optional

Reply:
SGTM